### PR TITLE
Standardize Package

### PR DESCRIPTION
**Pull request by maotian06**
# Description
Add the License section to `README.md` and add the license file based on @tyamaguchi-ucla 's suggestion.

# Checklist

- [x] This PR **does *NOT* contain** Protected Health Information [(PHI)](https://ohrpp.research.ucla.edu/hipaa/). A repo may ***need to be deleted*** if such data is uploaded. <br> Disclosing PHI is a ***major problem***[^1] - Even ***a small leak can be costly***[^2].
  
- [x] This PR **does *NOT* contain** germline genetic data[^3], RNA-Seq, DNA methylation, microbiome or other molecular data[^4].

[^1]: [UCLA Health reaches $7.5m settlement over 2015 breach of 4.5m patient records](https://healthitsecurity.com/news/ucla-health-reaches-7.5m-settlement-over-2015-breach-of-4.5m)
[^2]: [The average healthcare data breach costs $2.2 million, despite the majority of breaches releasing fewer than 500 records.](https://www.ponemon.org/local/upload/file/Sixth%20Annual%20Patient%20Privacy%20%26%20Data%20Security%20Report%20FINAL%206.pdf)
[^3]: [Genetic information is considered PHI.](https://www.genome.gov/about-genomics/policy-issues/Privacy#:~:text=In%202013%2C%20as%20required%20by,genetic%20information%20for%20underwriting%20purposes.)
  [Forensic assays can identify patients with as few as 21 SNPs](https://www.sciencedirect.com/science/article/pii/S1525157817305962)
[^4]: [RNA-Seq](https://www.nature.com/articles/ng.2248), [DNA methylation](https://ieeexplore.ieee.org/document/7958619), [microbiome](https://www.pnas.org/doi/pdf/10.1073/pnas.1423854112), or other molecular data can be used to predict genotypes (PHI) and reveal a patient's identity.

- [x] This PR **does *NOT* contain** other non-plain text files, such as: compressed files, images (*e.g.* `.png`, .`jpeg`), `.pdf`, `.RData`, `.xlsx`, `.doc`, `.ppt`, or other output files.

_&emsp; To automatically exclude such files using a [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file, see [here](https://github.com/uclahs-cds/template-base/blob/main/.gitignore) for example._

- [x] I have read the [code review guidelines](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3187646/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List).

- [x] I have set up or verified the `main` branch protection rule following the [github standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3190380/GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.

- [x] The name of the branch is meaningful and well formatted following the [standards](https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3189956/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].
  
- [ ] I have added the major changes included in this pull request to the `CHANGELOG.md` under the next release version or unreleased, and updated the date.

### CODE REVIEW

File: README.md
Comment (tyamaguchi-ucla) [inline README.md:243]: (suggestion | non-blocking) Do you want to add yourself? @maotian06

File: README.md
Comment (tyamaguchi-ucla) [inline README.md:249]: **(style|non-blocking)** It would be nice to use a range (you can check the commit history)

File: LICENSE
Comment (tyamaguchi-ucla) [inline LICENSE:1]: **(style|non-blocking)**

I think we generally go with `LICENSE.md` https://github.com/uclahs-cds/template-R-package/blob/main/LICENSE.md

**Review by tyamaguchi-ucla — APPROVED** [review]
Generally looks good to me. Anything else to add? @dan-knight @uclahs-cds/infrastructure-wg ? 

- CICDs looks old compared to the template https://github.com/uclahs-cds/template-R-package/tree/main/.github/workflows

- Separate PR but we want to add tests.

File: README.md
Comment (yashpatel6) [inline README.md:243]: **comment (non-blocking)**: I think we've also generally decided to not include email addresses in the READMEs unless there's a reason to include them for R packages?

File: NEWS
Comment (tyamaguchi-ucla) [inline NEWS:10]: **(question)**
@maotian06 I'm not sure if we've standardized the format of the `NEWS` file in our R package repos but I think we can generally follow our established standards if no objections. https://keepachangelog.com/en/1.1.0/
Has Infra WG recently discussed this? @yashpatel6 @dan-knight ?

Comment (yashpatel6) [reply NEWS:10]: We haven't; I'm not sure if the format as is was determined based on any standard specific to R packages? @tyamaguchi-ucla @dan-knight

File: NEWS
Comment (maotian06) [reply NEWS:1]: I need to release a new version as we wrap up the iSubGen for final editing, which is requested by Paul. 
But since the previous version `1.0.2` was not released, so I can keep the version number, right? @dan-knight

File: NEWS
Comment (maotian06) [reply NEWS]: This was a little confusing to me. Let's wait until the infra team decides.
I updated the `UPDATE` to `CHANGED` as in the changelog.

**Review by dan-knight — COMMENTED** [review]
Make sure to add `LICENSE.md` to `.Rbuildignore`.

**Comment by dan-knight** [comment]
The `Maintainer` field is usually Paul. Let's confirm with @pboutros that we should change it to him.

I'd use the `@AuthorsR` format for the authors (see [CEV's DESCRIPTION file](https://github.com/uclahs-cds/package-CancerEvolutionVisualization/blob/dc0d7c5a8fac5976835e30b510023d1b1cdfde26/DESCRIPTION#L5-L10)). I'd recommend Paul to have the `cre` role as maintainer, Natalie to have `aut`, and you to have `ctb`. (See [this resource](https://journal.r-project.org/archive/2012-1/RJournal_2012-1_Hornik~et~al.pdf) on the author roles.)

**Comment by dan-knight** [comment]
I'll update this to `NEWS.md` format before we merge.

File: NEWS
Comment (tyamaguchi-ucla) [reply NEWS]: @dan-knight and I looked into this. I've tagged `v1.0.0` and `v1.0.1` from the corresponding commits. I think you guys double check and we can release them (GH release). They should correspond to the CRAN release as well.

For the next version, we can bundle up `v1.0.2` as Dan's updates didn't change iSubGen itself.

**Comment by maotian06** [comment]
@dan-knight @tyamaguchi-ucla I have implemented all the updates. Can you help me take a look?

File: metadata.yaml
Comment (yashpatel6) [reply metadata.yaml:5]: This will be discussed yes but we're likely going to do away with `metadata.yaml` soon in github repos as it's largely redundant

Comment (maotian06) [reply metadata.yaml:5]: Got it, thanks! Let me know when it needs to be updated!

File: NEWS.md
Comment (maotian06) [reply NEWS.md:13]: I don't know what updates were made. I think Natalie did it.

File: DESCRIPTION
Comment (dan-knight) [inline DESCRIPTION:7]: Shouldn't this be Natalie, not Adriana?

File: DESCRIPTION
Comment (dan-knight) [inline DESCRIPTION:8]: And you should probably put me last. I've only contributed to general package standardization stuff. Nothing functional.

File: NEWS.md
Comment (tyamaguchi-ucla) [reply NEWS.md:13]: @maotian06 Have you checked the commit history?

Comment (maotian06) [reply NEWS.md:13]: image
I don't think there is anything in the commit history. @tyamaguchi-ucla

**Review by dan-knight — APPROVED** [review]
LGTM!

**Review by yashpatel6 — APPROVED** [review]
Generally looks good to me!

**Review by tyamaguchi-ucla — APPROVED** [review]
Looks good!

**Pull request merged**
